src/createRetryFetch.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+const defaultMaxRetry = 3;
+
+function defaultBackoffStrategy(retryCount) {
+  return new Promise((resolve) => setTimeout(resolve, retryCount * 100));
+}
+
+function createRetryFetch(maxRetry = defaultMaxRetry,
```

Comment:
The 2 arguments are optional, what do you think about accepting an object instead like we do in https://github.com/americanexpress/fetch-enhancers/blob/main/src/createBrowserLikeFetch.js#L41 would that make this more extensible / flexible in the future if we want to add more arguments?

Reply:
I am fine with that and can make that change.

Reply:
To me looks more extensible but let's wait for more opinions before making the change 👍 

@PixnBits @JAdshead thoughts?

Reply:
I have arguments in my head both ways 😓